src/client/utils/validation.js

```diff
@@ -11,6 +11,11 @@ validator.addFormat('address', {
   validate: isAddress,
 })
 
+export const validateDAOAddress = validator.compile({
+  type: 'string',
+  format: 'address',
```

Comment:
It looks like we're not currently checking that the logged-in ethereum id is a member of the provided organization. Is that accurate?

<img width="443" alt="Screen Shot 2019-07-02 at 15 16 05" src="https://user-images.githubusercontent.com/221614/60540154-80411d80-9cdc-11e9-867f-05e14a35b06f.png">